### DISCUSSION TITLE
Migration to new Mac

### DISCUSSION THREAD

**Discussion by Blackjacx** [junegunn/fzf#3259] [Q&A] [answered]
Hey there 👋

first of all great work. I loke using FZF 👍 

But now I have to move over to a new Mac and don't want to loose my history. I'm using ZSH on macOS and have the `.zsh_history` copied over from my old Mac. That means history without fzf works great.

**My question now is:** Where does `fzf` store their database or what do I have to move over to the new machine?

I could not find anything regarding that topic.

Thanks in advance :)

**Comment by junegunn** [comment, marked as answer]
fzf is a filter program, it doesn't have a database. For zsh, CTRL-R binding runs `fc -rl 1` to get the command history. So make sure that it works as expected on the new machine.

https://github.com/junegunn/fzf/blob/6be855be6af102a0f89932e5752ce75aa9713108/shell/key-bindings.zsh#L100
